validator/lib/validateTileset.js

```diff
@@ -173,3 +184,103 @@ function validateTileHierarchy(root, tilesetDirectory) {
             return message;
         });
 }
+
+function checkBoundingVolume(tileBV, parentBV, tileTransform, parentTransform) {
+    var returnString = undefined;
+
+    if (defined(tileBV) && defined(parentBV) && (defined(tileBV.box) || defined(tileBV.sphere) || defined(tileBV.region)) && (defined(parentBV.box) || defined(parentBV.sphere) || defined(parentBV.region))) {
```

Comment:
Similar to above this check will not be needed, but a separate PR will check for the existence of bounding volumes.